token/services/interop/htlc/transaction.go

```diff
@@ -227,11 +227,11 @@ func (t *Transaction) Claim(wallet *token.OwnerWallet, tok *token2.UnspentToken,
 		return err
 	}
 	script := &Script{}
-	if owner.Type != ScriptTypeExchange {
-		return errors.New("invalid owner type, expected exchange script")
+	if owner.Type != ScriptType {
```

Comment:
Shall we call ScriptType HTLCScript?

Reply:
as Angelo said: "just ScriptType given that htlc is in the package name"

Reply:
In that case, shall we just call Script? since the type of the owner is Script?

Reply:
@KElkhiyaoui , what about renaming `Script` (that contains the actual data) to `HTLC` and `ScriptType` to `Type`?

Reply:
we could leave the name as is and check later if refactoring is actually needed.